docker/hive2-no-auth/Dockerfile

```diff
@@ -1,140 +1,100 @@
-
-ARG JDK_URL="http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jdk-8u131-linux-x64.rpm"
-ARG JDK_RPM="jdk-8u131-linux-x64.rpm"
-ARG JDK_PATH="/usr/java/jdk1.8.0_131"
 ARG MYSQL_CONNECTOR_URL="https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.49.tar.gz"
 
+# Install dependencies
 RUN \
     # disable the notoriuosly unstable EPEL repo...
     rm -rf /etc/yum.repos.d/epel* && \
-    yum install -y wget && \
-
-    # install and trim Oracle JDK
-    wget -nv --header "Cookie: oraclelicense=accept-securebackup-cookie" $JDK_URL && \
-    rpm -ivh $JDK_RPM && rm $JDK_RPM && \
-    rm -rf $JDK_PATH/*src.zip \
-           $JDK_PATH/lib/missioncontrol \
-           $JDK_PATH/lib/visualvm \
-           $JDK_PATH/lib/*javafx* \
-           $JDK_PATH/jre/lib/plugin.jar \
-           $JDK_PATH/jre/lib/ext/jfxrt.jar \
-           $JDK_PATH/jre/bin/javaws \
-           $JDK_PATH/jre/lib/javaws.jar \
-           $JDK_PATH/jre/lib/desktop \
-           $JDK_PATH/jre/plugin \
-           $JDK_PATH/jre/lib/deploy* \
-           $JDK_PATH/jre/lib/*javafx* \
-           $JDK_PATH/jre/lib/*jfx* \
-           $JDK_PATH/jre/lib/amd64/libdecora_sse.so \
-           $JDK_PATH/jre/lib/amd64/libprism_*.so \
-           $JDK_PATH/jre/lib/amd64/libfxplugins.so \
-           $JDK_PATH/jre/lib/amd64/libglass.so \
-           $JDK_PATH/jre/lib/amd64/libgstreamer-lite.so \
-           $JDK_PATH/jre/lib/amd64/libjavafx*.so \
-           $JDK_PATH/jre/lib/amd64/libjfx*.so && \
-    # install dockerize
-    wget -nv https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-        && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-        && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
+    # install tools
+    yum install -y wget which && \
+    # add bigtop repo
+    wget -nv -O /etc/yum.repos.d/bigtop.repo https://archive.apache.org/dist/bigtop/bigtop-1.5.0/repos/centos-8/bigtop.repo && \
+    yum install -y \
+        nano \
+        hive \
+        hive-server2 \
+        hive-hbase \
+        hive-metastore \
+        hadoop-conf-pseudo \
+        hadoop \
+        tez \
+        zookeeper-server \
+        openssh \
+        openssh-clients \
+        openssh-server \
+        java-1.8.0-openjdk \
+        python3-setuptools \
+        python3-pip \
+        mysql-server \
+        && \
     # cleanup
     yum -y clean all && rm -rf /tmp/* /var/tmp/*
 
-ENV JAVA_HOME $JDK_PATH/jre/
-ENV PATH="/usr/lib/hive/bin:${PATH}"
-
-# Setup CDH repo, pin the CDH distribution to a concrete version
-RUN wget -nv http://archive.cloudera.com/cdh5/one-click-install/redhat/6/x86_64/cloudera-cdh-5-0.x86_64.rpm \
-  && yum --nogpgcheck localinstall -y cloudera-cdh-5-0.x86_64.rpm \
-  && rm cloudera-cdh-5-0.x86_64.rpm \
-  && rpm --import http://archive.cloudera.com/cdh5/redhat/6/x86_64/cdh/RPM-GPG-KEY-cloudera \
```

Comment:
Cloudera repo is now paid and with password auth :/

Reply:
oh damn, I did not know that